phi-coder/data_core/cluster_layer_5_8/cluster_bus.py

```
@@ -0,0 +1,67 @@
+        return signal
+    def listen(self, tag, default=None):
+        """
+        Retrieve the latest broadcast matching the tag.
+        Returns the payload (if encoded) or fallback default.
+        """
+        for signal in reversed(self.log):
+            if signal.startswith(f"📡 Broadcast: {tag} → "):
+                try:
+                    payload = signal.split("→", 1)[1].strip()
+                    return eval(payload) if payload else default
+                except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)